stm_workshop/lib/utils/utils.hpp

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <stdio.h>
```

Comment:
`iostream`?

Reply:
`stdio.h` gives us the printf method so we are good

Reply:
Do we need this or is this an outdated method? Just that it also appears to be deprecated and was wondering if there is any real benefits of using this or instead replacing `printf` with `std::cout` or similar?

Reply:
To be fair though if this is either irrelevant or would take any real effort to change, it's just a tutorial so can just ignore it for now.

Reply:
Actually we don't even use ASSERT, might just remove it if we really don't. Will check this in the afternoon.